File: code-recipes/php/test/easyship/DownloadShippingLabelRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\DownloadShippingLabelRecipe;
use Test\util\RecipeTestCase;

class DownloadShippingLabelRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new DownloadShippingLabelRecipe();
        $this->responses = [
            "reports-getReport",
            "reports-getReportDocument",
            ""
        ];
    }
}

File: code-recipes/java/src/test/java/easyship/DownloadShippingLabelRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class DownloadShippingLabelRecipeTest extends RecipeTest {

    protected DownloadShippingLabelRecipeTest() {
        super(
                new DownloadShippingLabelRecipe(),
                List.of(
                    "reports-getReport",
                    "reports-getReportDocument",
                    ""
                )
        );
    }
}
